ProjectPlan.md

```diff
@@ -1 +1,32 @@
-Edit here
+# Project Admin Portal Core Features
+
+### Team Diddy (14) - Daniel, Felix, Ricardo, [Truong Dang](https://github.com/truongd3), Vidhi
+### Prof. McCann
+### CS 490-section number
+### Fall 2024
+
+- Technologies used
+  - Node.js
+  - JavaScript
+  - GitHub
+
+- Team Member Assignments
+  - API 1: Add New User Command → Daniel
+  - API 2: Remove User Command → Daniel
+  - API 3: Update User Role Command → Daniel
+  - API 4: View System Status Command → Vidhi
+  - API 5: View User Activity Logs Command → Felix
+  - API 6: Set System Alert Command → Felix
+  - API 7: View System Logs Command → Ricardo
+  - API 8: Update System Configuration Command → Ricardo
+  - API 9: Manage Service Command → Truong Dang
+  - API 10: Generate Usage Report Command → Truong Dang
+
+- Project Plan: https://github.com/VD-25/osiris-admin-portal/blob/main/ProjectPlan.md 
+Steps for each API: API's to be written in node.js
+with starting base repo:
+team member needs to clone repo and and npm init a node.js project
```

Comment:
- Add "-" at the beginning (similar for other bullet points)
- Add ` before and after "npm init"